src/test/Components/CreateImageWizard/CreateImageWizard.test.tsx

```diff
@@ -24,9 +24,10 @@ const testTile = async (tile: HTMLElement) => {
 
   tile.focus();
   await waitFor(() => user.keyboard(' '));
-  expect(tile).toHaveClass('pf-m-selected');
-  await waitFor(() => user.keyboard(' '));
-  expect(tile).not.toHaveClass('pf-m-selected');
+  expect(tile).toHaveClass('pf-m-selectable');
+  // remove the keyboard test since cards don't
+  // have a keydown event and tiles have been
+  // deprecated
```

Comment:
Understandable, but maybe this comment will not make sense out of this PR. I would omit it.